fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NamespaceTools.java

```diff
@@ -75,7 +76,13 @@ public static void validatePath(final Session session, final String path) {
                     try {
                         namespaceRegistry.getURI(prefix);
                     } catch (final NamespaceException e) {
-                        throw new FedoraInvalidNamespaceException("Prefix " + prefix + " has not been registered", e);
+                        if (segment.equals("fcr:backup")) {
```

Comment:
It's unfortunate that this logic is buried here. Is it not possible to do this check in `FedoraExport`?